File: PRAK304/PRAK304-2210817310015-RIDHANISETIADI.py
A=int(input("Masukkan bilangan cacah : "))
if (A>0 and A<10) :
    print("Satuan")
elif A==0:
    print("Nol")
elif A > 99 :
    print("input melebihi limit bilangan")
elif A>=20 and A<99:
    print("Puluhan")
elif A>=10 and A<=19:
    print("Belasan")

File: PRAK304/PRAK304-2210817310015-RIDHANISETIADI.c
#include<stdio.h>
int main()
{
    int  nilai;
    printf("Masukkan bilangan cacah :");
    scanf("%d",&nilai);
if (nilai>0 && nilai<10){
    printf("Satuan");
}else if (nilai==0){
    printf("Nol");
}else if (nilai > 99){
    printf("input melebihi limit bilangan");
}else if (nilai>=20 && nilai<99){
    printf("Puluhan");
}else if (nilai>=10 && nilai<=19){
    printf("Belasan");
}
}
